wiki/Twitter/pl.md

```diff
@@ -1,17 +1,24 @@
----
-outdated_translation: true
----
-
 # Twitter
 
-## My także tweetujemy!
+osu! posiada kilka kont na Twitterze; każde z nich pełni inne funkcje. Pomimo niewielkiej aktywności oferują one wystarczające informacje o tym, co dzieje się w świecie osu!.
+
+## Serwisy
+
+| Awatar | Nazwa | Opis |
+| :-: | :-: | :-- |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Informacje dotyczące problemów ze stroną lub [Bancho](/wiki/Bancho_(server)). |
+| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Pomoc w sprawach związanych z kontem oraz społecznością osu!. Działa dzięki [drużynie do pomocy w sprawach związanych z kontami](/wiki/People/The_Team/Account_support_team). |
+
+## Społeczność
+
+| Awatar | Nazwa | Opis |
+| :-: | :-: | :-- |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Oficjalne źródło newsów i ogłoszeń. |
+| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Humorystyczne rozluźnienie sytuacji, gdy coś pójdzie nie tak. |
+| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat) | Newsy, ogłoszenia, krótkie ankiety społecznościowe prowadzone przez [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (nieużywane przez [główny team osu!](/wiki/People/The_Team)). |
 
-osu! posiada kilka kont na Twitter; każde z nich pełni inne funkcje. Pomimo niewielkiej aktywności oferują one wystarczające informacje o tym, co dzieje się w świecie osu!.
+## Personalne
 
-| Awatar | Konto | Opis |
+| Awatar | Nazwa | Opis |
 | :-: | :-: | :-- |
-| ![Awatar konta osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Relacjonowanie najnowszych wydarzeń ze społeczności. |
-| ![Awatar konta osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Informacje dotyczące problemów ze stroną lub bancho. |
-| ![Awatar konta Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Niezwiązane ściśle z osu!, ale pozostawione dla osób zainteresowanych. |
-| ![Awatar konta osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Pomoc w sprawach związanych z kontem oraz społecznością osu!. |
-| ![Awatar konta BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Humorystyczne rozluźnienie sytuacji, gdy coś pójdzie nie tak. |
+| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Personalne konto na Twitterze [twórcy osu!](/wiki/People/peppy), które nie jest dokładnie połączone z samym osu!, ale nadal obejmuje jego znaczną część. |
```

Comment:
Not sure about this new description, I'd probably revert to previous as it read better (although it was a bit less literal). Alternatively, I can offer this middle ground:

```suggestion
| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Personalne konto na Twitterze [twórcy osu!](/wiki/People/peppy), które nie jest ściśle związane z osu!, lecz na którym często poruszane są tematy związane z grą. |
```

It takes a bit of literary freedom but the translations aren't meant to be 100% exact as I understand it, the primary goal is to express the underlying intentions of the source material.